entries/v/vy.no.json

```
@@ -1,15 +1,15 @@
 {
-    "Vy": {
-      "domain": "vy.no",
-      "contact": {
-        "facebook": "vygruppen",
-        "twitter": "VyNordic"
-      },
-      "keywords": [
-        "transport"
-      ],
-      "regions": [
-        "no"
-      ]
-    }
-  }
\ No newline at end of file
+  "Vy": {
+    "domain": "vy.no",
+    "contact": {
+      "facebook": "vygruppen",
+      "twitter": "VyNordic"
```

@VyNordic doesn't seem to be any official account. Use this one instead:
```suggestion
      "twitter": "vygruppen"
```